tests/Tests.Domain/Tests.Domain.csproj

```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Bogus" Version="22.1.2" />
-    <PackageReference Include="Elastic.Elasticsearch.Managed" Version="0.3.5" />
+    <PackageReference Include="Elastic.Elasticsearch.Managed" Version="0.4.1" />
```

Comment:
See https://github.com/elastic/elasticsearch-net-abstractions/pull/58 for details.